Check/FrontEnd/AdminTheme.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * @file
+ * Contains \SiteAudit\Check\FrontEnd\AdminTheme.
+ */
+
+/**
+ * Class SiteAuditCheckFrontEndAdminTheme.
+ */
+class SiteAuditCheckFrontEndAdminTheme extends SiteAuditCheckAbstract {
+  /**
+   * Implements \SiteAudit\Check\Abstract\getLabel().
+   */
+  public function getLabel() {
+    return dt('Admin Theme In Use');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getDescription().
+   */
+  public function getDescription() {
+    return dt('Displays the admin theme being used');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultFail().
+   */
+  public function getResultFail() {}
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultInfo().
+   */
+  public function getResultInfo() {
+    return dt('Admin theme in use is "@theme"', array(
```

Comment:
As previously requested, just display the information about the theme: `Adminimal (adminimal_theme 8.x-1.x-dev)` (don't need to say "Admin theme in use is")